DEPS

```diff
@@ -401,6 +401,12 @@ deps = {
   'src/third_party/pkg/file':
   Var('github_git') + '/google/file.dart.git' + '@' + '427bb20ccc852425d67f2880da2a9b4707c266b4', # 6.1.0
 
+  'src/third_party/pkg/image':
+  Var('github_git') + '/brendan-duncan/image.git' + '@' + '3.0.2',
+
+  'src/third_party/pkg/petitparser':
+  Var('github_git') + '/petitparser/dart-petitparser' + '@' + '4.1.0',
```

Comment:
These dependencies (and XML) could be painful to upgrade, which we'll eventually want to do. Can we just get rid of them?

It looks like we only use this to encode and write a PNG image. We should be able to do that via `dart:ui`.

Reply:
(This, of course, should be done in a separate PR, and I'm not terribly partial to whether it's done before or after this one lands)

Reply:
Filed

Reply:
Yeah, in this PR, I think I should leave the test logic as-is, but I am totally on board with cutting down on the dependencies.